### DISCUSSION TITLE
It's been an amazing time with this project Winston

### DISCUSSION THREAD

**Discussion by joep26020** [winstxnhdw/lc-hax#443] [General]
### What happened?

I just wanted to thank you Winston for all the work you put into this. It has been an enjoyable part of my year following this project and seeing it progress. Sadly, there's not much to do because all that's left is to A. wait for v50 (which will probably be another month since zeekers doesn't have a team) or B. fight against the anti-anti-kicks. 

### Current Commit Hash

51555555

### Injector

- [ ] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have confirmed that my anti-virus is not blocking any of the relevant programs
- [X] I have done my due diligence to check for similar issues

**Comment by winstxnhdw** [comment]
Thanks! It was fun building this project as well.

**Reply by D1GQ** [reply]
And it's been fun and a great learning experience for me for tributing in this project! And through my eyes I see way more stuff to add other than updates and fighting back against anti cheats, I see this more than a hack client, I see this as a client, so I can think of many improvements that could be added to the game with patching!

**Reply by winstxnhdw** [reply]
I am glad you feel this way. The problem with adding a patch, is that for each patch we use, we are also increasing the chance that the project silently breaks in each game update. That's why I am a lot more apprehensive about adding patches than commands/mods.
